nipoppy_cli/nipoppy/tabular/bagel.py

```diff
@@ -56,19 +68,26 @@ def check_status(cls, value: str):
         return value
 
     @model_validator(mode="after")
-    def check_bids_id(self):
-        """Generate default value for optional BIDS ID field."""
-        if self.bids_id is None:
-            self.bids_id = participant_id_to_bids_id(self.participant_id)
+    def validate_after(self):
+        """Check fields."""
+        check_participant_id(self.participant_id, raise_error=True)
+        check_session_id(self.session_id, raise_error=True)
+
+        if self.bids_participant is None:
+            self.bids_participant = participant_id_to_bids_participant(
+                self.participant_id
+            )
+        if self.session is None:
+            self.session = session_id_to_bids_session(self.session_id)
 
 
 class Bagel(BaseTabular):
     """A file to track data availability/processing status."""
 
     # column names
     col_participant_id = "participant_id"
-    col_bids_id = "bids_id"
-    col_session = "session"
+    col_bids_participant = "bids_participant"
+    col_session_id = "session_id"
```

Comment:
this if probably minor and doesn't affect anything, but in some other files you also name this as `bids_session_id`?

Reply:
I don't see `bids_session_id` anywhere, but I do use `bids_session` (instead of `session`) now in other parts of the code. The bagel file is a bit trickier because we still want things to be compatible with the dashboard, so cannot rename all columns immediately.